cdc/owner.go

```diff
@@ -1395,12 +1417,12 @@ func (o *Owner) run(ctx context.Context) error {
 		return errors.Trace(err)
 	}
 
-	err = o.handleAdminJob(ctx)
+	err = o.calcResolvedTs(ctx)
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	err = o.calcResolvedTs(ctx)
+	err = o.handleAdminJob(ctx)
```

Comment:
Is it necessary to change the order of these two operations?

Reply:
I think even without changing the order, it will work correctly.